pkg/apiserver/apiserver.go

```diff
@@ -151,7 +153,7 @@ func (s *Service) Start(ctx context.Context) error {
 			info.RegisterRouter,
 			clusterinfo.RegisterRouter,
 			profiling.RegisterRouter,
-			profiling.RegisterConprofRouter,
+			conprof.RegisterRouter,
```

Comment:
Maybe RegisterRouter can also be declared in the `conprof.Module`, like this:
```
var Module = fx.Options(
	fx.Provide(newService),
	fx.Invoke(registerRouter),
)
```

Reply:
Let me try.